cpu/Makefile.include.cortexm_common

```diff
@@ -6,7 +6,7 @@ export USEMODULE += cortexm_common
 # Export the peripheral drivers to be linked into the final binary:
 export USEMODULE += periph
 # all cortex MCU's use newlib as libc
-export USEMODULE += newlib
+export USEMODULE += newlib_thread_safe
```

Comment:
I restore this line and make the application include it using the pseudo module?